### ISSUE TITLE
Make `ok`s green

### ISSUE THREAD

**Issue by CIAvash** [Raku/tap-harness6#45]
Is it possible to make the `ok` words for each successful test green?

**Comment by CIAvash** [comment]
Or maybe the whole `ok` line. Same for `not ok` with whole line being red.

**Comment by Leont** [comment]
Yeah that is possible, and sounds like a reasonable idea (though I think it makes more sense to only mark the ok/not ok in color).

**Comment by Leont** [comment]
> I think it makes more sense to only mark the ok/not ok in color

That's exactly what I ended up doing.
